tests/testthat/test-glmnetUtils.R

```diff
@@ -0,0 +1,63 @@
+context("glmnetUtils")
+
+skip_if_not_installed("modeltests")
+library(modeltests)
+
+skip_if_not_installed("glmnetUtils")
+library(glmnet)
```

Comment:
These tests don't use the `glmnetUtils` package or a formula interface?

Reply:
Done. None of the relevant tests fail.